### ISSUE TITLE
Functions of the Hue Smart Button (ROM001)

### ISSUE THREAD

**Issue by dirkg173** [sebbebebbe/com.philips.hue.zigbee#296]
Hi @JohanBendz,
the Smart Button has the same functions as the Dimmer Switch:
- On-button pressed
- Off-button pressed
- Dim buttons (Dim-up pressed / Dim-up hold / Dim-down pressed / Dim-down hold / Dim-button released)

I guess this is a mistake.
The following functions should be correct, shouldn't they?
- Button pressed
- Button hold
- Button released (maybe)

Best regards,
Dirk

**Comment by JohanBendz** [comment]
Hi @dirkg173, yes this is a misstake on my side. The code somehow got mixed up. I will fix asap.

**Comment by JohanBendz** [comment]
Hi @dirkg173.
I'v edited the code, it was a mess ;)

Added a few cards that will be in next update. As I removed cards that was faulty you will have to fix any flows using the device.

New cards:
  "Button pressed"
  "Button pressed on (odd times)"
  "Button pressed off (even times)"
  "Button held (any time)"
  "Button held (<1 sec)"
  "Button held (>1 sec)"
  "Button held - up"
  "Button held - down"
  "Button released (if status=on and held)"
